### ISSUE TITLE
One more badge type

### ISSUE THREAD

**Issue by KOLANICH** [repology/repology#387]
The badge contain 3 rows. The top one has the best available version number in its most left cell, the rest of cells are distros families logos, such as "Arch, Debian-based, Ubuntu-based, Gentoo, Fedora, RHEL, *BSD-based, Solaris-based". The second row has the best stable version as its first cell, the rest of cells correspond to stable versions found in repos.
The third row is for devel versions. The rows are appropriately colored. The set of families and repos in families should be configurable.

**Comment by AMDmi3** [comment]
I don't think this would be useful, because even if repository grouping is implemented (which I've had in plans), there are still too many of them (and there will be even more), and I don't see much point in showing only partial information (besides, we have a set of per-repository badges for this).

Also the form factor doesn't look practical - you don't generally want long solid horizontal object because page width may vary in a huge range (from mobile devices to 4K screens), and the growing horizontal badge will not fit in some cases and leave a lot of free space in others. This is solved by using a set of smaller badges which are subject to wrapping.

There are a lot of possibilities to shrink the vertical badge in both directions:
- implement autoscaling based on font metrics
- implement repo grouping
- implement line wrapping for long repo names
- implement tunable font size

**Comment by KOLANICH** [comment]
Mobile devices are assumed to be used in horisontal orientation to browse the web.
Thie proposed badge is assummed to consume the 3 rows in the Readme.md after rendering.
But the vertical badge takes lots of rows, so it cannot be pasted into the beginning of Readme.md - the traditional location for badges and shields.

**Comment by AMDmi3** [comment]
Big badge may be used with floating (badges page has code and [libversion](https://github.com/repology/libversion) is an example) so regardless of how tall it is, it'll not produce any empty space and will fit organically. I still don't see any use in horizontal badge, however it seems feasible to implement is as a set of small 3 row per-repository badges - this is both naturally customizable and handles line wrapping.

**Comment by XVilka** [comment]
Were there any updates on this? Still as relevant as years ago.

**Comment by AMDmi3** [comment]
There were no updates. This issue has lowest priority and I still doubt its usefulness.
